packages/constellation/src/components/Compositions/Table/AdvancedTableExample/AdvancedTableExample.stories.tsx

```diff
@@ -33,7 +33,7 @@ const Template: ComponentStory<FC<{ onClick: () => void }>> = ({ onClick }) => {
           const sort = header.column.getIsSorted()
           const onClick = header.column.getToggleSortingHandler()
 
-          return <TextCell text='Asset' isHeader sort={sort} toggleSort={onClick} />
```

Comment:
isHeader is being removed from the cells here as they are not required when the Row components has the `header` prop